joomla/joomla-cms
# [4.0] Namespace com_login

Pull Request for Issue # .

### Summary of Changes
This simple PR convert com_login component to namespace MVC for Joomla 4.

### Testing Instructions
1. Install latest Joomla 4.0-dev with sample data, then apply patch (or better install my branch https://github.com/joomdonation/joomla-cms/archive/namespace_com_login.zip )

2. Try to login to administrator area of the site, make sure it is working as expected.

3. Try to logout from administrator area of the site, make sure it is working as expected.
